src/components/index.ts

```diff
@@ -38,7 +35,7 @@ export { default as ProposalDetail } from './HistoryTables/MsigProposals/Proposa
 export * from './HistoryTables/ActorState'
 export * from './HistoryTables/PendingMsgContext'
 export * from './HistoryTables/methodName'
-export { LineWrapper, Line, Parameters } from './HistoryTables/detail'
```

Comment:
Feels like we might as well export the new lines component somewhere in here since they'll presumably be useful to us down the road? doesn't add any weight to the package anyway?

Reply:
It is exported in `src\components\Layout\index.js`